src/main/steps/applicant2/how-to-apply-financial-order/content.ts

```diff
@@ -0,0 +1,21 @@
+import { TranslationFn } from '../../../app/controller/GetController';
+import { FormContent } from '../../../app/form/Form';
+import {
+  form as applicant1Form,
+  generateContent as applicant1GenerateContent,
+} from '../../applicant1/how-to-apply-financial-order/content';
+
+export const form: FormContent = {
```

Comment:
can just remove this and return applicant1 stuff if there is nothing special being added.